src/userLog.js

```diff
@@ -5,7 +5,7 @@ var Helper = require("./helper");
 
 module.exports.write = function(user, network, chan, msg) {
 	try {
```

Comment:
Can we move this whole try/catch block into main `exports`, so that it's not invoked on each `write`? Alternatively this can be moved all the way to `index` CLI where `config` is created.